src/components/SearchModal/SearchModal.js

```diff
@@ -615,7 +614,7 @@ const SearchModal = ({
                 isMultiSelect={isMultiSelect}
                 searchType={searchType}
                 containerRef={ref => {
-                  if (ref && isTitleSearch) {
+                  if (ref) {
```

Comment:
Could you please investigate should we fix eslint error or add eslint-disable for this rule for lines?

Reply:
Thanks, disabled the warning for now